core/assets/bundles/bundle_nl.properties

```diff
@@ -1060,13 +1060,13 @@ unit.eradicator.name = Eradicator
-tutorial.drill = Met de hand delven is inefficient.\n[accent]Drills []kunnen automatisch voor je delven.\nPlaats er een op de koper.
-tutorial.drill.mobile = Met de hand delven is inefficient.\n[accent]Drills []kunnen automatisch voor je delven.\nZoek de drill rechts onderin.\nSelecter de[accent] mechanische drill[].\nPlaats het op de koper door erop te klikken, druk dan op het[accent] vinkje[] om het bouwen te bevestigen.\nKlik op de[accent] X knop[] om het te anuleren.
-tutorial.blockinfo = Elk blok heeft andere statistieken. Elke drill kan enkel bepaalde dingen delven.\nOm het van een blok te checken,[accent] druk op de "?" knop terwijl je het blok vast hebt.[]\n\n[accent]Lees de stats van de Mechanische Drill maar eens.[]
-tutorial.conveyor = [accent]Lopende Banden[] worden gebruikt om je items naar je core te krijgen.\nLeg een line aan van je drills tot aan je core.
-tutorial.conveyor.mobile = [accent]Lopende Banden[] worden gebruikt om je items naar je core te krijgen.\nLeg een line aan van je drills tot aan je core.\n[accent] Doe dit door je vinger een paar seconden stil te houden[] en dan in een richting te slepen.\n\n[accent]{0}/{1} lopende banden in 1x geplaatst\n[accent]0/1 items afgeleverd
+tutorial.intro = Welkom bij de[scarlet] Mindustry Tutorial.[]\nBegin met het[accent] mijnen van koper[]. Klik op een vakje met koper om het te mijnen.\n\n[accent]{0}/{1} koper
+tutorial.intro.mobile = Welkom bij de[scarlet] Mindustry Tutorial.[]\nSleep over het scherm om te bewegen.\n[accent]Knijp met 2 vingers [] om in en uit te zoomen.\nBegin met het[accent] mijnen van koper[]. Beweeg dichterbij, en klik er dan op.\n\n[accent]{0}/{1} koper
+tutorial.drill = Met de hand mijnen is inefficient.\n[accent]Boren []kunnen automatisch voor je mijnen.\nPlaats een boor op koper.
+tutorial.drill.mobile = Met de hand mijnen is inefficient.\n[accent]Boren []kunnen automatisch voor je mijnen.\nZoek de boren rechts onderin.\nSelecter de[accent] mechanische boor[].\nPlaats het op de koper door erop te klikken, druk dan op het[accent] vinkje[] om het bouwen te bevestigen.\nKlik op de[accent] X knop[] om het te anuleren.
+tutorial.blockinfo = Elk blok heeft andere statistieken. Elke boor kan enkel bepaalde dingen mijnen.\nOm het van een blok te checken,[accent] druk op de "?" knop terwijl je het blok vast hebt.[]\n\n[accent]Lees de statatistieken van de Mechanische Boor maar eens.[]
+tutorial.conveyor = [accent]Lopende Banden[] worden gebruikt om je items naar je core/basis te krijgen.\nLeg een lijn aan van je boren tot aan je core/basis.
+tutorial.conveyor.mobile = [accent]Lopende Banden[] worden gebruikt om je items naar je core/basis te krijgen.\nLeg een lijn aan van je boren tot aan je core/basis.\n[accent] Doe dit door je vinger een paar seconden stil te houden[] en dan in een richting te slepen.\n\n[accent]{0}/{1} Lopende banden worden in 1x geplaatst\n[accent]0/1 items afgeleverd
```

Comment:
> Lopende banden worden in 1x geplaatst

ze "worden" niet geplaatst, het is een teller qua hoeveel lopende banden in 1x worden geplaatst in dezelfde mobiele bouw-actie 🤔

Reply:
"Lopende Banden in 1x geplaatst" Is volgens mij niet echt een kloppende zin als je kijkt naar het engels origineel

Reply:
> het is een teller qua hoeveel lopende banden in 1x worden geplaatst in dezelfde mobiele bouw-actie 🤔

En ze worden toch allemaal geplaatst als je hebt gesleept en loslaat...

Reply:
blijkbaar is de engelse variant een tijdje terug veranderd, maar is de oude engelse versie in de nl bundle blijven hangen, die vervolgens vertaald is 🤔 https://github.com/Anuken/Mindustry/commit/0f79298c282a06c5fbfc69b17c3c9d9c6c846396

Reply:
lijkt dus wel alsof `{0}/{1}` gewoon weg kan eruit, want er zit geen teller meer in het origineel 😗